packages/pdf-export-server/.npmrc

```diff
@@ -0,0 +1 @@
+scripts-prepend-node-path=true
```

Comment:
I don't know if this is needed?

Reply:
I've seeen it in other packages, thought that's the default setting. Do you suggest we remove it?

Reply:
Maybe something I added.... it's ok will clean up in a separate PR